pytorch_lightning/trainer/trainer.py

```diff
@@ -483,8 +483,8 @@ def __init__(
         # init flags for SLURM+ddp to work
         self.proc_rank = 0
         self.world_size = 1
-        self.node_rank = 0
         self.configure_slurm_ddp(self.num_nodes)
+        self.node_rank = self.determine_node_rank()
```

Comment:
seems like this function is not defined?

Reply:
@williamFalcon it is! in the `distributed_data_parallel.py` mixin.

Reply:
Ah, typo. Correcting.

Reply:
oh cool.. ic.

I guess the names don't match
determine_ddp_node_rank -> determine_node_rank

tests are failing there

Reply:
```suggestion
        self.node_rank = self. determine_ddp_node_rank()
```